src/porepy/models/constitutive_laws.py

```diff
@@ -3947,6 +3966,47 @@ def stiffness_tensor(self, subdomain: pp.Grid) -> pp.FourthOrderTensor:
         mu = self.solid.shear_modulus() * np.ones(subdomain.num_cells)
         return pp.FourthOrderTensor(mu, lmbda)
 
+    def characteristic_contact_traction(
+        self, subdomains: list[pp.Grid]
+    ) -> pp.ad.Operator:
+        """Characteristic traction [Pa].
+
+        The value is computed from the solid constants and the characteristic
+        displacement. Inversion of this relationship, i.e.,
+        u_char=u_char(t_char), is possible if the characteristic traction is
+        easier to estimate than the characteristic displacement.
+
+        Parameters:
+            subdomains: List of subdomains where the characteristic traction is defined.
+
+        Returns:
+            Scalar operator representing the characteristic traction.
+
+        """
+        size = Scalar(np.max(self.domain.side_lengths()))
+        strain = self.characteristic_displacement(subdomains) / size
+        t_char = strain * self.youngs_modulus(subdomains)
+        t_char.set_name("characteristic_contact_traction")
+        return t_char
+
+    def characteristic_displacement(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
+        """Characteristic displacement [m].
+
+        The value is fetched from the solid constants. See also the method
```

Comment:
I guess we should consider a separate data structure for charactecistic quantities, but not until we are ready to roll this out throughout the code.

Reply:
Yes, see TODO at line 326 of material_constants.py.